src/components/programs/spark/projectcard.tsx

```diff
@@ -0,0 +1,41 @@
+import { FaSquareGithub } from "react-icons/fa6";
```

Comment:
```suggestion
import { FaGithub } from "react-icons/fa";
```
use this one instead the square version makes me sad